src/main/java/uk/ac/cam/cl/dtg/isaac/api/EventsFacade.java

```diff
@@ -1034,6 +1041,11 @@ public final Response createBookingForMe(@Context final HttpServletRequest reque
                     .toResponse();
             }
 
+            if (EventStatus.CANCELLED.equals(event.getEventStatus())) {
+                return new SegueErrorResponse(Status.BAD_REQUEST, "Sorry this event is cancelled. Please try again later.")
```

Comment:
Although I doubt anyone will take this literally, I don't think "please try again later" makes much sense here. Most events probably won't be un-cancelled once cancelled.

Reply:
Good point 😆 I'll remove it